subprojects/internal-testing/internal-testing.gradle.kts

```diff
@@ -27,6 +27,7 @@ dependencies {
     compile(library("groovy"))
 
     compile(project(":baseServices"))
+    compile(project(":core"))
```

Comment:
Can we avoid this by putting `AsmConstants` in `base-services`?

Reply:
yep